### PR TITLE
Adding swag to repo

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Adding swag to repo + some introduction for project. To be moved to docs later.

@jstaffor can you take look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+![Logo](img/logo-blue.png)
+> Workforce management platform
 
-Feedhenry RainCatcher Core Repository
+[![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core.svg?branch=master)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core) [![Coverage Status](https://coveralls.io/repos/github/feedhenry-raincatcher/raincatcher-core/badge.svg?branch=master)](https://coveralls.io/github/feedhenry-raincatcher/raincatcher-core?branch=master)
 
 ## About RainCatcher
 
-## Developing
+RainCatcher is an Workforce Management platform with main focus on delivering set of modules and user interface for mobile developers trying to integrate WFM features into their application.
```
Comment (wtrocki) [inline README.md]: We may add more industry specific info from here: 
https://en.wikipedia.org/wiki/Workforce_management

That's probably going to be added directly into docs

**Comment by wtrocki** [comment]
Render https://github.com/wtrocki/raincatcher-core/blob/6d8f5ee93eb9e9691892757420f9c3ca8284326f/README.md

File: README.md (hunk 1 above)
Comment (JameelB) [inline README.md]: RainCatcher is `a` Workforce Management platform with `the` main focus on delivering `a` set of modules and user interface for mobile developers `who would like` to integrate WFM features into their application.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,39 @@
...
+
+RainCatcher consist of the multiple layers:
```
Comment (JameelB) [inline README.md]: remove `the` or use 'consist of `the following` layers' instead.

File: README.md [hunk 3]
```diff
@@ -17,9 +42,8 @@ Check your Node.JS version running `node -v`
 ```bash
 git clone https://github.com/feedhenry-raincatcher/raincatcher-core.git
 cd raincatcher-core
-npm install -g lerna
 npm install
-lerna bootstrap
+npm bootstrap
```
Comment (JameelB) [inline README.md]: should this not be `npm run bootstrap?`

File: README.md [hunk 4]
```diff
@@ -1,14 +1,39 @@
...
+![Logo](img/logo-blue.png)
+> Workforce management platform
 
-Feedhenry RainCatcher Core Repository
+[![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core.svg?branch=master)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core) [![Coverage Status](https://coveralls.io/repos/github/feedhenry-raincatcher/raincatcher-core/badge.svg?branch=master)](https://coveralls.io/github/feedhenry-raincatcher/raincatcher-core?branch=master)
 
 ## About RainCatcher
 
-## Developing
+RainCatcher is a Workforce Management platform with the main focus on delivering a set of modules and user interface for mobile developers who would like to integrate WFM features into their application.
```
Comment (jstaffor) [inline README.md]: Change "would like" to "want"

Comment (jstaffor) [inline README.md]: Should "RainCatcher" variable be used in this repo - I don't know?

File: README.md [hunk 5]
```diff
@@ -1,14 +1,39 @@
...
+
+## Getting Started
+
+To run demo application please follow instructions from
+our [Reference User Interface](https://github.com/feedhenry-raincatcher/raincatcher-angularjs) repository.
+
+## Developing core components
 
-RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
+RainCatcher supports and is tested on node v 6.x LTS.
```
Comment (jstaffor) [inline README.md:36]: Should "node v 6.x" variable be used in this repo - I don't know?

File: README.md [hunk 6]
```diff
@@ -1,14 +1,39 @@
...
 
-Feedhenry RainCatcher Core Repository
+[![Build Status](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core.svg?branch=master)](https://travis-ci.org/feedhenry-raincatcher/raincatcher-core) [![Coverage Status](https://coveralls.io/repos/github/feedhenry-raincatcher/raincatcher-core/badge.svg?branch=master)](https://coveralls.io/github/feedhenry-raincatcher/raincatcher-core?branch=master)
 
 ## About RainCatcher
 
-## Developing
+RainCatcher is a Workforce Management platform with the main focus on delivering a set of modules and user interface for mobile developers who would like to integrate WFM features into their application.
+
+RainCatcher consist of the following layers:
```
Comment (jstaffor) [inline README.md]: "Layers"? should it be "Components", "Artefacts"

Comment (wtrocki) [reply README.md]: Yeah that was hard to name considering that points bellow mentioning components itself. How about `areas`?

File: README.md (hunk 4 above)
Comment (wtrocki) [reply README.md]: that's markdown. We going to move this to documentation and replace.

File: README.md (hunk 5 above)
Comment (wtrocki) [reply README.md:36]: As above. We going to move this to variable when pulling out to docs. 
We do not have variable support in markdown :(

File: README.md (hunk 6 above)
Comment (jstaffor) [reply README.md]: `areas` +1

**Review by JameelB — APPROVED** [review]

**Pull request merged**